package.json

```diff
@@ -10,23 +10,27 @@
     "test": "jest"
   },
   "dependencies": {
+    "eslint": "^8.42.0",
```

Comment:
Jak już majstrujesz, to powypierdalaj "^" sprzed wszystkich paczek, przez to w yarn.lock smród i syf jest później